_web/_site.yml

```diff
@@ -27,5 +27,7 @@ navbar:
           href: manova2.html
         - text: "Split-Plot ANOVA"
           href: split-plot.html
+        - text: "LDA"
+          href: LDA.html
     - text: "About"
       href: about.html
```

Comment:
It is generally bad practice to not have a newline at the end of the file.

Reply:
I might have placed the comment to the wrong file.

Reply:
I'm not sure what you mean. What comment? What wrong file?

Reply:
In the GitHub file compare, there is always a complainant on any file not ending with a newline.  But we can address newline issues in a different PR.

I guess I did post (what I meant by comment) to the right line of code with this issue (the file is right).